src/components/VRadio/VRadio.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <label :for="id" class="radio-label relative flex leading-5">
+    <input
+      :id="id"
+      v-bind="$attrs"
+      :value="value"
+      class="radio appearance-none relative flex-shrink-0 bg-white w-5 h-5 border border-dark-charcoal rounded-full me-3 transition-colors focus:outline-none focus:ring focus:ring-offset-2 focus:ring-primary disabled:bg-dark-charcoal-10 disabled:border-dark-charcoal-40"
+      type="radio"
+      :checked="isChecked"
+      @input="handleInput"
+    />
+    <Radiomark
+      class="radiomark absolute start-0 w-5 h-5 text-dark-charcoal opacity-0 transition-opacity"
+      focusable="false"
+      width="20"
+      height="20"
+      role="img"
```

Comment:
Tbh I copied this bit from `VCheckbox`. Since the preceding `input` is fairly accessible to screenreaders, should we go one step further and `aria-hidden` this SVG entirely?

Reply:
It would be nice to set `aria-hidden` to both the radiomark and the checkbox checkmark.